src/js/targeted-content.js

```diff
@@ -128,6 +128,14 @@ function initTargetedContentFor(el) {
     if (matchEl) {
       setState(matchEl, 'closed');
     }
+
+    const elTop = el.getBoundingClientRect().top;
```

Comment:
Does something like `el.scrollIntoView({ behavior: "smooth", block: "center" })` work instead of needing to calculate a specific position?

Reply:
Probably - was going for something that doesn't need another polyfill.

Reply:
Cool, as far as I know that shouldn't need a polyfill. I think it's the `smooth` behaviour that isn't always available, but that's true with `scrollTo` as well I think? Looks like calling `scrollIntoView` on the button / heading of targeted content might do the job. Crack on if I've misunderstood though.

Reply:
Just went for boring old feature detection.  I think the niceness of smooth scrolling and the extra bit of space between the top of the targeted content is worth it.